src/Rest/Routes/GeolocationCountriesRoute.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * The class register route for getting Geolocation list of countries endpoint.
+ *
+ * @package EightshiftForms\Rest\Routes
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Rest\Routes;
+
+use EightshiftForms\Exception\UnverifiedRequestException;
+use EightshiftForms\Geolocation\GeolocationInterface;
+
+/**
+ * Class GeolocationCountriesRoute
+ */
+class GeolocationCountriesRoute extends AbstractBaseRoute
+{
+	/**
+	 * Instance variable of ClientInterface data.
+	 *
+	 * @var GeolocationInterface
+	 */
+	protected $geolocation;
+
+	/**
+	 * Create a new instance that injects classes
+	 *
+	 * @param GeolocationInterface $geolocation Inject GeolocationInterface which holds Geolocation data.
+	 */
+	public function __construct(GeolocationInterface $geolocation)
+	{
+		$this->geolocation = $geolocation;
+	}
+
+	/**
+	 * Get the base url of the route
+	 *
+	 * @return string The base URL for route you are adding.
+	 */
+	protected function getRouteName(): string
+	{
+		return '/geolocation-countries';
+	}
+
+	/**
+	 * Returns allowed methods for this route.
+	 *
+	 * @return string
+	 */
+	protected function getMethods(): string
+	{
+		return static::READABLE;
+	}
+
+	/**
+	 * Get callback arguments array
+	 *
+	 * @return array<string, mixed> Either an array of options for the endpoint, or an array of arrays for multiple methods.
+	 */
+	protected function getCallbackArguments(): array
+	{
+		return [
+			'methods' => $this->getMethods(),
+			'callback' => [$this, 'routeCallback'],
+			'permission_callback' => [$this, 'permissionCallback'],
```

Comment:
Where is the `permissionCallback` callback?

Reply:
in the parent abstract class